.ci/test.sh

```diff
@@ -8,6 +8,13 @@ set -exuo pipefail
 EXTRA_DEPS="$1"
 MARKERS="$2"
 
+# Integration test settings
+export WANDB_ENTITY='mosaicml-public-integration-tests'
+export WANDB_PROJECT='integration-tests'
```

Comment:
Does each test create a run?  Wondering if we should have a separate project with a unique ID or timestamp for each invocation of the test suite if that's the case.

Reply:
Only tests marked as `@remote` that use wandb will create a run. Good point -- will create a seperate project for each build

Reply:
Resolved.